src/ui/components/atoms/label-articulos-destacados/LabelArtDestacados.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import LabelArtDestacados from './LabelArtDestacados';
+import '../../../../globals/_variables.scss';
+
+const meta: Meta<typeof LabelArtDestacados> = {
+  title: 'ui/components/atoms/label-articulos-destacados',
```

Comment:
please rename the name use English

Reply:
buenas tardes ingeniero, ya corregi lo que me dijo